planner/core/task.go

```diff
@@ -522,7 +522,6 @@ func (p *PhysicalHashJoin) attach2Task(tasks ...task) task {
 	lTask := finishCopTask(p.ctx, tasks[0].copy())
 	rTask := finishCopTask(p.ctx, tasks[1].copy())
 	p.SetChildren(lTask.plan(), rTask.plan())
-	p.schema = BuildPhysicalJoinSchema(p.JoinType, p)
```

Comment:
Shall we also remove this in `planner/cascades/implementation/join.go/HashJoinImpl.AttachChildren`?

Reply:
done.